File: 30daysOFcode_Php/day 25.php
<?php
$_fp = fopen("php://stdin", "r");
/* Enter your code here. Read input from STDIN. Print output to STDOUT */
$T=(int)readline();
function isPrime($n){
    if($n==2){
        return true;
    }
    else if($n%2==0){
        return false;
    }
    else{
        if($n==1 || $n<=0){
            return false;
        }
        else{
            for($j=3;$j*$j<=$n;$j+=2){
                if($n%$j===0){
                    return false;
                }
            }
            return true;
        }
    }
}
for($i=0;$i<$T;$i++){
    $a=(int)readline();
    $stat=isPrime($a);
    if($stat){
        echo("Prime\n");
    }
    else{
        echo("Not prime\n");
    }
}
?>

File: 30daysOFcode_Swift/day 25.swift
// Enter your code here 
func findPrime(input:Int) -> String {
    if input <= 1 {
        return "Not prime"
    }
    if input <= 3 {
        return "Prime"
    }
    var i = 2
    while i*i <= input {
        if input % i == 0 {
            return "Not prime"
        }
        i = i + 1
    }
    return "Prime"
}

var t = Int(readLine()!)!

for _ in 0...t-1 {
    print(findPrime(Int(readLine()!)!))
}
